config/types.go

```diff
@@ -36,6 +36,10 @@ const (
 	// Name of the dataplane's container
 	ConsulDataplaneContainerName = "consul-dataplane"
 
+	// Indicates that a service was registered by a control plane container running in
+	// a mesh task managed by Consul dataplane
+	DataplaneBasedMeshTaskTag = "dataplane-based-mesh-task"
```

Comment:
Chose such a name to avoid exposing the `agentless` to the public. Open for feedback on this.

Reply:
Yeah. That makes more sense. I just wanted to be extra cautious on making sure the controller is truly backward compatible. Looks like the node name should be enough to ensure compat.